docs/main/docs/executor/executor-overview.md

```diff
@@ -59,17 +59,17 @@ We'll therefore focus on describing the orders handling process from EVMs (Ether
 4. Set your keys for dedicated networks in `.env` file (see `.env.example` for reference)
 
 ```
-export ARBITRUM_SEPOLIA_PRIVATE_KEY=0x0e5b64068e340f44bc99a47951685f96051d28d6f23a9e8c527f64a8c53691d7
-export SCROLL_SEPOLIA_PRIVATE_KEY=0x0e5b64068e340f44bc99a47951685f96051d28d6f23a9e8c527f64a8c53691d7
-export BINANCE_TESTNET_PRIVATE_KEY=0x0e5b64068e340f44bc99a47951685f96051d28d6f23a9e8c527f64a8c53691d7
-export SEPOLIA_PRIVATE_KEY=0x0e5b64068e340f44bc99a47951685f96051d28d6f23a9e8c527f64a8c53691d7
-export BASE_GOERLI_PRIVATE_KEY=0x0e5b64068e340f44bc99a47951685f96051d28d6f23a9e8c527f64a8c53691d7
-export BASE_SEPOLIA_PRIVATE_KEY=0x0e5b64068e340f44bc99a47951685f96051d28d6f23a9e8c527f64a8c53691d7
-export OPTIMISM_SEPOLIA_PRIVATE_KEY=0x0e5b64068e340f44bc99a47951685f96051d28d6f23a9e8c527f64a8c53691d7
+export ARBITRUM_SEPOLIA_PRIVATE_KEY=PRIVATE-KEY-HERE
+export SCROLL_SEPOLIA_PRIVATE_KEY=PRIVATE-KEY-HERE
+export BINANCE_TESTNET_PRIVATE_KEY=PRIVATE-KEY-HERE
+export SEPOLIA_PRIVATE_KEY=PRIVATE-KEY-HERE
+export BASE_GOERLI_PRIVATE_KEY=PRIVATE-KEY-HERE
+export BASE_SEPOLIA_PRIVATE_KEY=PRIVATE-KEY-HERE
+export OPTIMISM_SEPOLIA_PRIVATE_KEY=PRIVATE-KEY-HERE
 
 export ENABLED_NETWORKS='base-sepolia,optimism-sepolia,binance-testnet,scroll-sepolia,arbitrum-sepolia'
 export RPC_HEALTH_CHECK_INTERVAL_SEC=3
-export PRICER_PROVIDER_URL='https://rpc.ankr.com/multichain/ae45090060327148428269ea88b3801903f4c81f8a700d8f04dc99a6cb3cb7ac'
+export PRICER_PROVIDER_URL='https://rpc.ankr.com/multichain/API-KEY-HERE'
```

Comment:
`PRICER_PROVIDER_URL` is no longer used
`PRICER_USE_MULTICHAIN` is no longer used

please sync with our current state so its as minimal as possible

Reply:
I think this whole strategy that Maciej published (which these keys are part of) needs to be revisited. Better to remove sensitive data now, and then take time to sync with current state?

Reply:
The key was already rotated. Private keys are from some random generated wallet which doesn't hold any tokens.
Lets tackle both issues, removing unused variables and sensitive data in this quick PR.
Writing those comments is taking us longer than it would take to remove a couple of variables.